### PR TITLE
Fix Zawgyi normalization mistake, adding tests

### LINKED ISSUE #34: Transliterate Z Norm error
Current behavior

class : TransliterateZNorm.java
method : convert 

input (zawgyi)
မၪၨဴရီ

Expected behavior
output (zawgyi)
မၪၨဴရီ

Actual Result
output (zawgyi)
မညၨဴရီ

### PR DESCRIPTION
**Pull request by sven-oly**
Fixes #34, adding test cases.

### CODE REVIEW

File: clients/js/spec/zawgyi_converter_spec.js [hunk 1]
```diff
@@ -14,14 +14,14 @@
...
     z: " ",
     u: " "
-},
-{
-    z: "123",
-    u: "123"
-},
+  },
+  // {
+  //   z: "123",
```
[the remark this answers is not in the document]
Comment (sven-oly) [reply clients/js/spec/zawgyi_converter_spec.js]: Reverted.

File: clients/js/spec/zawgyi_converter_spec.js [hunk 2]
```diff
@@ -743,16 +759,30 @@ describe("ZawgyiConverter U2Z", function () {
     });
 });
 
-describe("Zawgyi to Unicode Conversion Compatibility Test", function() {
```
[the remark this answers is not in the document]
Comment (sven-oly) [reply clients/js/spec/zawgyi_converter_spec.js]: Thanks for catching this. I've restored.

File: clients/js/spec/zawgyi_converter_spec.js [hunk 3]
```diff
@@ -14,14 +14,10 @@
...
-{
+  {
     z: " ",
     u: " "
-},
-{
-    z: "123",
-    u: "123"
-},
+  },
```
Comment (sffc) [inline clients/js/spec/zawgyi_converter_spec.js]: Please restore these lines.

File: clients/js/spec/zawgyi_converter_spec.js [hunk 4]
```diff
@@ -743,16 +755,43 @@ describe("ZawgyiConverter U2Z", function () {
...
+            var test_case = zawgyi_normalize_tests[i];
+            var expected = test_case.zn;
+            var actual = converter.normalizeZawgyi(test_case.z);
+            expect(i+" "+actual).toEqual(i+" "+expected);
+        }
+
+    });
+});
+
+describe("Zawgyi Normalization Test", function() {
```
Comment (sffc) [inline clients/js/spec/zawgyi_converter_spec.js]: You added this twice?

File: clients/js/spec/zawgyi_converter_spec.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (sffc) [reply clients/js/spec/zawgyi_converter_spec.js]: It still shows as deleted for me

**Pull request merged**